manifests/kustomize/test/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
My initial intention was to keep it here, so that when people change manifests they can also easily discover how test infra is using an overlay and not break it.

I get your point of not being distracting now, because for most people coming here, they are only looking for lightweight deployment. I'm convinced, will move it to test/ folder.

Reply:
Done